src/historywork/GetHistoryArchiveStateWork.cpp

```diff
@@ -61,6 +61,8 @@ GetHistoryArchiveStateWork::doWork()
 
     else
     {
+        CLOG(INFO, "History")
```

Comment:
This log is not useful when we download the latest checkpoint, it just prints `0`. Instead, it should use the name that it calculated in the next line.